src/main/webapp/app/shared/sidebar/sidebar-accordion/sidebar-accordion.component.html

```diff
@@ -1,46 +1,48 @@
 @for (groupKey of Object.keys(groupedData); track groupKey; let i = $index) {
-    <div
-        id="test-accordion-item-container-{{ i }}"
-        class="pointer"
-        [ngClass]="{ 'd-none': (!showAddOptions || searchValue?.length !== 0) && !(groupedData[groupKey].entityData | searchFilter: ['title', 'type'] : searchValue)?.length }"
-    >
+    @if ((groupedData[groupKey].entityData | searchFilter: ['title', 'type'] : searchValue)?.length! > 0 || sidebarItemShowAlways()![groupKey]) {
```

Comment:
Can we solve this without non null assertions but instead explicit checks before?

Reply:
The second one i can, the first one i sadly cannot. For this i would need to refactor the component, which is already done by [#9356](https://github.com/ls1intum/Artemis/pull/9356). This should only be a quick fix